src/module/skb/bpf.rs

```diff
@@ -310,3 +311,21 @@ pub(super) fn unmarshal_dev(
 
     Ok(())
 }
+
+/// Net namespace information retrieved from SKBs.
+#[derive(Default)]
+#[repr(C, packed)]
+struct SkbNsEvent {
+    /// Net namespace id.
+    netns: u32,
+}
+unsafe impl Plain for SkbNsEvent {}
+
+pub(super) fn unmarshal_ns(
+    raw_section: &BpfRawSection,
+    fields: &mut Vec<EventField>,
+) -> Result<()> {
+    let event = parse_event::<SkbNsEvent>(raw_section)?;
+    fields.push(event_field!("netns", event.netns));
```

Comment:
Should we have a netns num-to-name resolution in userspace? The number will probably little to the user